src/index.js

```diff
@@ -100,8 +101,40 @@ const middleware = (config = {}) => (req, res, next) => {
         error: 'Signature verification failed!'
       })
     }
+    const certificates = JSON.parse(req.headers['x-authrite-certificates']).map(async cert => {
+      if(cert.subject !== identityKey) {
+        return res.status(401).json({
+          error: 'Certificate subject does not match identity key of the request sender!'
```

Comment:
For all errors, we need to use the Standard Format for Babbage Errors, which is on Confluence. https://projectbabbage.atlassian.net/wiki/spaces/PB/pages/8486913/Standard+Formats+for+Babbage+Errors?atlOrigin=eyJpIjoiNDNkMjk0MzgzM2ZlNDUxYjgwMWY2NmQxMDM3MWM3ODciLCJwIjoiaiJ9

Effectively, this means using `status`, `code`, and `description`. The `status` should be `error`, the `code` should be an error code in the machine-readable format `ERR_UNIQUE_CODE`, and `description` should be a human-readable string.